app/views/entities/_form.html.erb

```diff
@@ -0,0 +1,26 @@
+<%= form_with(model: @entity, url: category_entities_path, method: :post, local: true) do |form| %>
+  <% if entity.errors.any? %>
+    <div style="color: red">
+      <h2><%= pluralize(entity.errors.count, "error") %> prohibited this entity from being saved:</h2>
+      <ul>
+        <% entity.errors.each do |error| %>
+          <li><%= error.full_message %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
+  <div>
+    <%= form.label :name, style: "display: block" %>
+    <%= form.text_field :name %>
+  </div>
+  <div>
+    <%= form.label :amount, style: "display: block" %>
+    <%= form.number_field :amount %>
+  </div>
+  <div>
+    <%= form.hidden_field :user_id, :value => current_user.id %>
+  </div>
+  <div>
+    <%= form.submit %>
+  </div>
```

Comment:
- [ ] On "Add a new transaction" page, The user should be able to choose the categories to which the transaction belongs. Please add a section to your form where the user can select the categories. Check out the requirements below 👇:

The user fills out a form to create a new transaction with:
* name (mandatory)
* amount (mandatory)
* categories (mandatory at least one)The user fills out a form to create a new transaction with: